staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2444,6 +2444,12 @@ type Container struct {
 	// +optional
 	// +listType=atomic
 	ResizePolicy []ContainerResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
+	// Restart policy for the container.
+	// Always is the only supported value.
+	// It MUST be set only for init containers.
+	// +featureGate=SidecarContainers
+	// +optional
+	RestartPolicy *RestartPolicy `json:"restartPolicy,omitempty" protobuf:"bytes,24,opt,name=restartPolicy"`
```

Comment:
Don't need pointer for this string. A string with omitempty will is optional and absent when the value is "".